app/models/level.rb

```diff
@@ -0,0 +1,17 @@
+# Level
+class Level < ApplicationRecord
+  # has_many :users | need when connecting with the User model
+  # Explain (  level.id is a user_level  )
```

Comment:
bad practice to write the code in advance if you do not need it not

Reply:
remove these comments

Reply:
fixed.